src/index.html

```diff
@@ -0,0 +1,183 @@
+                type="text"
+                id="input-name"
+                placeholder="Type your name"
+                class="modal__input"
+                required
+                minlength="2"
+                maxlength="40"
+              />
+              <span id="input-name-error" class="modal__error"></span>
+            </label>
+            <label for="input-description" class="modal__label">
+              Description
+              <input
+                type="text"
+                id="input-description"
+                placeholder="Description"
+                class="modal__input"
+                required
+                minlength="2"
+                maxlength="200"
+              />
+              <span id="input-description-error" class="modal__error"></span>
+            </label>
+            <button type="submit" class="modal__submit-btn">Save</button>
+          </form>
+        </div>
+      </div>
+      <div class="modal" id="add-profile-modal">
+        <div class="modal__container">
+          <button type="button" class="modal__close-btn"></button>
+          <h2 class="modal__title">New post</h2>
+          <form name="add-image" class="modal__form" novalidate>
+            <label for="add-card-link-input" class="modal__label">
+              Image link
+              <input
+                type="url"
+                id="add-card-link-input"
+                placeholder="Place a link to the picture"
+                class="modal__input"
+                required
+              />
+              <span id="add-card-link-input-error" class="modal__error"></span>
+            </label>
+            <label for="input-caption" class="modal__label">
+              Caption
+              <input
+                type="text"
+                id="input-caption"
+                placeholder="Type your caption"
+                class="modal__input"
+                required
+                minlength="2"
+                maxlength="30"
+              />
+              <span id="input-caption-error" class="modal__error"></span>
+            </label>
+            <button type="submit" class="modal__submit-btn modal__submit-btn-add">Save</button>
+          </form>
+        </div>
+      </div>
+    </div>
+    <div class="modal" id="preview-modal">
+      <div class="modal__container modal__container_type_preview">
+        <button
+          class="modal__close-btn modal__close-btn_type_preview"
+          type="button"
+        ></button>
+        <img src="" alt="" class="modal__image" />
+        <p class="modal__caption"></p>
+      </div>
+    </div>
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button type="button"
+        class="modal__close-btn">
+      </button>
+        <h2 class="modal__title">Edit profile avatar</h2>
+        <form name="edit-avatar-form"
+        class="modal__form"
+        novalidate>
+          <label for="input-avatar-name"
```

Comment:
` for="input-avatar-name"`

It's not about the `name`. It should be `profile-avatar-input` here (like the `id`)


```js
          <label for="profile-avatar-input"

```